recipes/mysql-connector-c/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+
+
+
+class MysqlConnectorCConan(ConanFile):
+    name = "mysql-connector-c"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "A MySQL client library for C development."
+    topics = ("conan", "mysql", "sql", "connector", "database")
+    homepage = "https://dev.mysql.com/downloads/connector/c/"
+    author = "Bincrafters <bincrafters@gmail.com>"
```

Comment:
now is not that important, but in the future we will probably remove authors from recipes as there will be contributions by other people, so the authors will be tracked in the github repo